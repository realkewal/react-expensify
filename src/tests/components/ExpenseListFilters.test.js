import React from 'react';
import 'react-dates/initialize';
import {shallow} from 'enzyme';
import { ExpenseListFilters } from '../../components/ExpenseListFilters';
import { filters, altFilters } from '../fixtures/filters';

let setTextFilter, sortByDate, sortByAmount, setStartDate, setEndDate, wrapper;

beforeEach(() => {
    // setTextFilter = jest.fn();
    // sortByDate = jest.fn();
    // sortByAmount = jest.fn();
    // setStartDate = jest.fn();
    // setEndDate = jest.fn();
    // wrapper = shallow(
    //     <ExpenseListFilters 
    //         filters={filters}
    //         setTextFilter={setTextFilter}
    //         sortByDate={sortByDate}
    //         sortByAmount={sortByAmount}
    //         setStartDate={setStartDate}
    //         setEndDate={setEndDate}
    //     />
    // );
});


test('should render expense list filters correctly', () => {
    // expect(wrapper).toMatchSnapshot();
});

test('should render expense list filters with alt data correctly', () => {
    // wrapper.setProps({
    //     filters: altFilters
    // });
    // expect(wrapper).toMatchSnapshot();
});

test('should handle text change', () => {

});

test('should sort by date', () => {

});

test('should sort by amount', () => {

});

test('should handle date changes', () => {

});

test('should handle date focus changes', () => {

});