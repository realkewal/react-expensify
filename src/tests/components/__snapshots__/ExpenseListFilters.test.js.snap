// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expense list filters correctly 1`] = `undefined`;
